### PR TITLE
feat: add more languages

### PR DESCRIPTION
**Pull request by anna-geller**
EE: https://github.com/kestra-io/kestra-ee/pull/1501/files

### CODE REVIEW

File: ui/src/translations/ru.json [hunk 1]
```diff
@@ -0,0 +1,813 @@
...
+      }
+    },
+    "override": {
+      "title": "Вы собираетесь перезаписать текущий flow",
+      "details": "Предыдущий flow можно будет восстановить через вкладку редакций."
+    },
+    "is deprecated": "устарело",
+    "success": "Успех",
+    "save": "Сохранить",
+    "save task": "Сохранить Task",
```
Comment (smunteankestra) [inline ui/src/translations/ru.json]: should be 
"save task": "Сохранить задачу"

Comment (anna-geller) [inline ui/src/translations/ru.json]:
```suggestion
    "save task": "Сохранить задачу",
```

File: ui/src/translations/ru.json [hunk 2]
```diff
@@ -0,0 +1,813 @@
...
+    "Per page": "на страницу",
+    "new": "Новый",
+    "edit": "Редактировать",
+    "edit flow": "Редактировать flow",
+    "executions": "Выполнения",
+    "execution": "Выполнение",
+    "details": "Детали",
+    "overview": "Обзор",
+    "gantt": "Гантт",
+    "logs": "Logs",
```
Comment (smunteankestra) [inline ui/src/translations/ru.json]:
```suggestion
    "logs": "Логи",
```

File: ui/src/translations/ru.json [hunk 3]
```diff
@@ -0,0 +1,813 @@
...
+    "duration": "Длительность",
+    "running duration": "Время выполнения",
+    "queued duration": "Время в очереди",
+    "choose file": "Выберите файл или перетащите его сюда...",
+    "launch execution": "Выполнить",
+    "warning flow with triggers": "Этот flow содержит triggers, и ручное выполнение завершится неудачей, если этот flow зависит от trigger выражений.",
+    "triggered": "Execution trigger",
```
Comment (smunteankestra) [inline ui/src/translations/ru.json]:
```suggestion
    "triggered": "Активирован trigger",
```

File: ui/src/translations/ru.json [hunk 4]
```diff
@@ -0,0 +1,813 @@
...
+      "manage desc": "Управление сохраненными поисковыми фильтрами",
+      "save filter": "Сохранить фильтр",
+      "filter name": "Имя фильтра",
+      "filters": "Фильтры"
+    },
+    "steps": "Шаги",
+    "state": "Состояние",
+    "search term in message": "Поиск по термину в сообщении",
+    "search": "Поиск",
+    "search blueprint": "Поиск blueprints",
```
Comment (smunteankestra) [inline ui/src/translations/ru.json:137]:
```suggestion
    "search blueprint": "Поиск чертежей",
```

File: ui/src/translations/ru.json [hunk 5]
```diff
@@ -0,0 +1,813 @@
...
+    "all tags": "Все теги",
+    "source search": "Поиск по источнику",
+    "filter by log level": "Фильтр по уровню log",
+    "selected": "Выбрано",
+    "task": "Task",
+    "task logs": "Task logs",
```
Comment (smunteankestra) [inline ui/src/translations/ru.json]:
```suggestion
    "task logs": "Журналы задач",
```

File: ui/src/translations/ru.json [hunk 6]
```diff
@@ -0,0 +1,813 @@
...
+    "steps": "Шаги",
+    "state": "Состояние",
+    "search term in message": "Поиск по термину в сообщении",
+    "search": "Поиск",
+    "search blueprint": "Поиск blueprints",
+    "all tags": "Все теги",
+    "source search": "Поиск по источнику",
+    "filter by log level": "Фильтр по уровню log",
+    "selected": "Выбрано",
+    "task": "Task",
```
Comment (smunteankestra) [inline ui/src/translations/ru.json]:
```suggestion
    "task": "Задача",
```

File: ui/src/translations/ru.json [hunk 7]
```diff
@@ -0,0 +1,813 @@
...
+    "start datetime": "Дата и время начала",
+    "end datetime": "Дата и время окончания",
+    "restart": "Перезапуск",
+    "restart latest revision": "Перезапустить последнюю редакцию",
+    "restart tooltip": "Перезапустить выполнение с <code>{state}</code> task",
+    "restarted": "Выполнение перезапущено",
+    "restart confirm": "Вы уверены, что хотите перезапустить выполнение <code>{id}</code>?",
+    "restart change revision": "Вы можете изменить редакцию, которая будет использоваться для нового выполнения.",
+    "replay": "Повтор",
+    "replay from task tooltip": "Создать аналогичное выполнение, начиная с task <code>{taskId}</code>",
```
Comment (smunteankestra) [inline ui/src/translations/ru.json]:
```suggestion
    "replay from task tooltip": "Создать аналогичное выполнение, начиная с задачи <code>{taskId}</code>",
```

File: ui/src/translations/ru.json [hunk 8]
```diff
@@ -0,0 +1,813 @@
...
+    "change status confirm": "Вы уверены, что хотите изменить статус task <code>{task}</code> для выполнения <code>{id}</code>?",
+    "change status hint": {
+      "WARNING": "['Flow будет помечен как WARNING.', 'Следующие tasks будут выполнены.', 'Tasks с ошибками будут выполнены.']",
+      "FAILED": "['Flow будет помечен как FAILED.', 'Другие tasks не будут выполнены.', 'Tasks с ошибками будут выполнены.']",
+      "SUCCESS": "['Flow перезапустится, так как этот task был успешным.', 'Все заблокированные tasks будут выполнены.', 'Flow будет в состоянии SUCCESS, если все запуски tasks будут успешными.']",
+      "RUNNING": "['Flow перезапустится и выполнит все следующие tasks.', 'Все заблокированные tasks будут выполнены.']"
+    },
+    "mark as": "Пометить как <code>{status}</code>",
+    "kill": "Убить",
+    "kill parents and subflow": "Убить родителей и subflows",
```
Comment (smunteankestra) [inline ui/src/translations/ru.json]:
```suggestion
    "kill parents and subflow": "Убить родителей и подпроцессы",
```

but this sounds a bit disturbing - maybe in community people will contribute to something better

File: ui/src/translations/ru.json [hunk 9]
```diff
@@ -0,0 +1,813 @@
...
+    "outputs": "Выводы",
+    "output": "Вывод",
+    "eval": {
+      "title": "Отладочные выводы",
+      "tooltip": "Выполните любое выражение Pebble и проверьте контекст выполнения."
+    },
+    "attempt": "Попытка",
+    "toggle output display": "Переключить отображение вывода",
+    "name": "Имя",
+    "key": "Key",
```
Comment (smunteankestra) [inline ui/src/translations/ru.json]:
```suggestion
    "key": "Ключ",
```

File: ui/src/translations/ru.json [hunk 10]
```diff
@@ -0,0 +1,813 @@
...
+    "value": "Value",
```
Comment (smunteankestra) [inline ui/src/translations/ru.json]:
```suggestion
    "value": "Значение",
```

File: ui/src/translations/ru.json [hunk 11]
```diff
@@ -0,0 +1,813 @@
...
+    "trigger refresh": "Обновить",
+    "refresh": "Обновить",
+    "topology-graph": {
+      "graph-orientation": "Ориентация графика",
+      "zoom-in": "Увеличить",
+      "zoom-out": "Уменьшить",
+      "zoom-reset": "Сбросить масштаб",
+      "zoom-fit": "Подогнать"
+    },
+    "show task logs": "Показать task logs",
```
Comment (smunteankestra) [inline ui/src/translations/ru.json]:
```suggestion
    "show task logs": "Показать логи задачи",
```

File: ui/src/translations/ru.json [hunk 12]
```diff
@@ -0,0 +1,813 @@
...
+    "show task outputs": "Показать task выводы",
```
Comment (smunteankestra) [inline ui/src/translations/ru.json]:
```suggestion
    "show task outputs": "Показать выводы задачи",
```

File: ui/src/translations/ru.json [hunk 13]
```diff
@@ -0,0 +1,813 @@
...
+    "show task source": "Показать исходный код task",
```
Comment (smunteankestra) [inline ui/src/translations/ru.json]:
```suggestion
    "show task source": "Показать исходный код задачи",
```

File: ui/src/translations/ru.json [hunk 14]
```diff
@@ -0,0 +1,813 @@
...
+    "specific task": "Конкретный task",
```
Comment (smunteankestra) [inline ui/src/translations/ru.json]:
```suggestion
    "specific task": "Конкретная задача",
```

File: ui/src/translations/ru.json [hunk 15]
```diff
@@ -0,0 +1,813 @@
...
+    "display output for specific task": "Показать вывод для конкретного task",
```
Comment (smunteankestra) [inline ui/src/translations/ru.json]:
```suggestion
    "display output for specific task": "Показать вывод для конкретной задачи",
```

File: ui/src/translations/ru.json [hunk 16]
```diff
@@ -0,0 +1,813 @@
...
+    "display metric for specific task": "Показать метрику для конкретного task",
```
Comment (smunteankestra) [inline ui/src/translations/ru.json]:
```suggestion
    "display metric for specific task": "Показать метрику для конкретной задачи",
```

File: ui/src/translations/ru.json [hunk 17]
```diff
@@ -0,0 +1,813 @@
...
+    "display direct sub tasks count": "Показать количество прямых подзадач",
+    "stream": "Поток",
+    "execution statistics": "Статистика выполнения",
+    "stats": "Статистика",
+    "your usage": "Ваше использование",
+    "namespaces": "Namespaces",
+    "tasks": "Tasks",
```
Comment (smunteankestra) [inline ui/src/translations/ru.json]:
```suggestion
    "tasks": "Задачи",
```

File: ui/src/translations/ru.json [hunk 18]
```diff
@@ -0,0 +1,813 @@
...
+    "password length constraint": "Пароль не должен превышать 256 символов",
+    "passwords do not match": "Пароли не совпадают",
+    "avg duration": "Средняя длительность выполнения",
+    "neutral trend": "Стабильно",
+    "up trend": "Увеличивается",
+    "down trend": "Уменьшается",
+    "update aborted": "обновление прервано",
+    "invalid flow": "Неверный flow",
+    "invalid yaml": "Неверный YAML",
+    "inputs": "Inputs",
```
Comment (smunteankestra) [inline ui/src/translations/ru.json]:
```suggestion
    "inputs": "Входные данные",
```

File: ui/src/translations/ru.json [hunk 19]
```diff
@@ -0,0 +1,813 @@
...
+    "input": "Input",
```
Comment (smunteankestra) [inline ui/src/translations/ru.json]:
```suggestion
    "input": "Ввод",
```

File: ui/src/translations/ru.json [hunk 20]
```diff
@@ -0,0 +1,813 @@
...
+    "template creation": "Создание шаблона",
+    "templates": "Шаблоны",
+    "templates deprecated": "Шаблоны устарели. Пожалуйста, используйте subflows вместо них. См. <a href=\"https://kestra.io/docs/migration-guide/0.11.0/templates\" target=\"_blank\">раздел миграции</a>, объясняющий, как можно мигрировать с шаблонов на subflows.",
+    "no result": "Нет результатов для текущего выбора",
+    "trigger": "Trigger",
+    "triggers": "Triggers",
+    "trigger details": "Подробности trigger",
+    "triggerflow disabled": "Trigger отключен в определении flow",
+    "conditions": "условия",
+    "taskruns": "Запуски tasks",
```
Comment (smunteankestra) [inline ui/src/translations/ru.json]:
```suggestion
    "taskruns": "Запуски задач",
```

File: ui/src/translations/ru.json (hunk 8 above)
Comment (anna-geller) [reply ui/src/translations/ru.json]: yeah, we need to be careful about the wording here. German translation was initially quite brutal too so I had to soften it 

I think your "Kill parents and subprocesses" sounds great!

File: ui/src/translations/ru.json [hunk 21]
```diff
@@ -0,0 +1,813 @@
...
+    "Fold auto": "Редактор: автоматическое сворачивание многострочных строк",
+    "Fold content lines": "Свернуть многострочные строки",
+    "Unfold content lines": "Развернуть многострочные строки",
+    "Max displayable": "Максимально отображаемое",
+    "Total": "Всего",
+    "sub flow": "Subflow",
```
Comment (smunteankestra) [inline ui/src/translations/ru.json]:
```suggestion
    "sub flow": "Подпроцесс",
```

File: ui/src/translations/ru.json [hunk 22]
```diff
@@ -0,0 +1,813 @@
...
+        "title": "Неавторизован",
+        "content": "Вам нужно пройти аутентификацию для доступа к этой странице."
+      },
+      "403": {
+        "title": "Доступ запрещен",
+        "content": "У вас нет необходимых прав для доступа к этой странице."
+      }
+    },
+    "back_to_dashboard": "Вернуться на панель управления",
+    "copy logs": "Копировать logs",
```
Comment (smunteankestra) [inline ui/src/translations/ru.json]:
```suggestion
    "copy logs": "Копировать логи",
```

File: ui/src/translations/ru.json [hunk 23]
```diff
@@ -0,0 +1,813 @@
...
+    "download logs": "Скачать logs",
```
Comment (smunteankestra) [inline ui/src/translations/ru.json]:
```suggestion
    "download logs": "Скачать логи",
```

File: ui/src/translations/ru.json [hunk 24]
```diff
@@ -0,0 +1,813 @@
...
+    "delete logs": "Удалить logs",
```
Comment (smunteankestra) [inline ui/src/translations/ru.json]:
```suggestion
    "delete logs": "Удалить логи",
```

File: ui/src/translations/ru.json [hunk 25]
```diff
@@ -0,0 +1,813 @@
...
+    "delete_all_logs": "Вы уверены, что хотите удалить все logs?",
```
Comment (smunteankestra) [inline ui/src/translations/ru.json]:
```suggestion
    "delete_all_logs": "Вы уверены, что хотите удалить все логи?",
```

File: ui/src/translations/ru.json [hunk 26]
```diff
@@ -0,0 +1,813 @@
...
+    "toggle fullscreen": "Переключить полноэкранный режим",
+    "copied": "Скопировано",
+    "tags": "Теги",
+    "disabled flow title": "Этот flow отключен",
+    "disabled flow desc": "Этот flow отключен, пожалуйста, включите его для выполнения.",
+    "label": "Label",
```
Comment (smunteankestra) [inline ui/src/translations/ru.json]:
```suggestion
    "label": "Метка",
```

File: ui/src/translations/ru.json [hunk 27]
```diff
@@ -0,0 +1,813 @@
...
+    "labels": "Метки",
+    "label filter placeholder": "Метка как 'key:value'",
+    "execution labels": "Метки выполнения",
+    "wrong labels": "Пустой key или value не допускается в метках",
+    "feeds": {
+      "title": "Что нового в Kestra"
+    },
+    "delete execution running": "<div class=\"alert alert-warning mt-2 mb-0\">Это выполнение все еще RUNNING, удаление его не остановит.<br />Вам нужно убить выполнение, чтобы остановить его.</div>",
```
Comment (smunteankestra) [inline ui/src/translations/ru.json]:
```suggestion
    "delete execution running": "<div class=\"alert alert-warning mt-2 mb-0\">Это выполнение все еще выполняется, удаление его не остановит.<br />Вам нужно убить выполнение, чтобы остановить его.</div>",
```

File: ui/src/translations/ru.json [hunk 28]
```diff
@@ -0,0 +1,813 @@
...
+    "homeDashboard": {
+      "title": "Панель управления",
+      "today": "Сегодня",
+      "yesterday": "Вчера",
+      "last28Days": "Последние 28 дней",
+      "lastXdays": "Последние {days} дней",
+      "namespacesExecutions": "Выполнения по namespace",
+      "namespacesErrorExecutions": "Ошибки выполнения по namespace",
+      "failedExecutions": "Неудачные выполнения",
+      "errorLogs": "Ошибки logs",
```
Comment (smunteankestra) [inline ui/src/translations/ru.json]:
```suggestion
      "errorLogs": "Логи ошибок",
```

File: ui/src/translations/ru.json [hunk 29]
```diff
@@ -0,0 +1,813 @@
...
+    "export all flows": "Экспортировать все flows",
+    "import": "Импорт",
+    "disable": "Отключить",
+    "enable": "Включить",
+    "enabled": "Включено",
+    "template delete": "Вы уверены, что хотите удалить <code>{templateCount}</code> шаблон(ы)?",
+    "flow delete": "Вы уверены, что хотите удалить <code>{flowCount}</code> flow(ы)?",
+    "templates deleted": "<code>{count}</code> Шаблон(ы) удалены",
+    "flows deleted": "<code>{count}</code> Flow(ы) удалены",
+    "flow disable": "Вы уверены, что хотите отключить <code>{flowCount}</code> flow(ы)?",
```
Comment (smunteankestra) [inline ui/src/translations/ru.json:403]: this is a really good translation flow(ы) - meaning the plural of flows (adapted to russian!) this is a very "russian" way to translate

File: ui/src/translations/ru.json [hunk 30]
```diff
@@ -0,0 +1,813 @@
...
+    "Next step": "Следующий шаг",
+    "Previous step": "Предыдущий шаг",
+    "Finish": "Завершить",
+    "Step": "Шаг",
+    "welcome aboard": "🚀 Добро пожаловать в Kestra!",
+    "welcome aboard content": "Используйте наше Руководство, чтобы создать ваш первый flow и ознакомьтесь с Blueprints для поиска дополнительных примеров.",
+    "welcome display require": "Запустите ваш <strong>первый flow</strong>, чтобы начать",
+    "welcome button create": "Создать мой первый flow",
+    "live help": "Живая помощь",
+    "show task documentation": "Показать документацию task",
```
Comment (smunteankestra) [inline ui/src/translations/ru.json]:
```suggestion
    "show task documentation": "Показать документацию задачи",
```

File: ui/src/translations/ru.json [hunk 31]
```diff
@@ -0,0 +1,813 @@
...
+    "hide task documentation": "Скрыть документацию task",
```
Comment (smunteankestra) [inline ui/src/translations/ru.json]:
```suggestion
    "hide task documentation": "Скрыть документацию задачи",
```

File: ui/src/translations/ru.json [hunk 32]
```diff
@@ -0,0 +1,813 @@
...
+    "show task documentation in editor": "Показать документацию task в редакторе",
```
Comment (smunteankestra) [inline ui/src/translations/ru.json]:
```suggestion
    "show task documentation in editor": "Показать документацию задачи в редакторе",
```

File: ui/src/translations/ru.json [hunk 33]
```diff
@@ -0,0 +1,813 @@
...
+    "show documentation": "Показать документацию",
+    "focus task": "Нажмите на любой элемент, чтобы увидеть его документацию.",
+    "validate": "Проверить",
+    "add global error handler": "Добавить глобальный обработчик ошибок",
+    "add error handler": "Добавить обработчик ошибок",
+    "add trigger": "Добавить trigger",
+    "edit metadata": "Редактировать метаданные",
+    "taskDefaults": "Task по умолчанию",
```
Comment (smunteankestra) [inline ui/src/translations/ru.json]:
```suggestion
    "taskDefaults": "Задача по умолчанию",
```

File: ui/src/translations/ru.json [hunk 34]
```diff
@@ -0,0 +1,813 @@
...
+    "disabled": "Отключено",
+    "before": "до",
+    "after": "после",
+    "add at position": "Добавить {position} <code>{task}</code>",
+    "create first task": "Создайте ваш первый task",
```
Comment (smunteankestra) [inline ui/src/translations/ru.json]:
```suggestion
    "create first task": "Создайте свою первую задачу",
```

File: ui/src/translations/ru.json [hunk 35]
```diff
@@ -0,0 +1,813 @@
...
+    "dynamic": "Динамический",
+    "choice": "Выбор",
+    "sequential": "Последовательный",
+    "can not delete": "Невозможно удалить",
+    "can not have less than 1 task": "Каждый flow должен иметь хотя бы один task.",
```
Comment (smunteankestra) [inline ui/src/translations/ru.json]:
```suggestion
    "can not have less than 1 task": "Не может быть менее одного задания",
```

File: ui/src/translations/ru.json [hunk 36]
```diff
@@ -0,0 +1,813 @@
...
+    "task id already exists": "Task Id уже существует",
+    "Task Id already exist in the flow": "Task Id {taskId} уже существует в flow.",
+    "task id": "Task ID",
```
Comment (smunteankestra) [inline ui/src/translations/ru.json]:
```suggestion
    "task id": "ID задачи",
```

File: ui/src/translations/ru.json [hunk 37]
```diff
@@ -0,0 +1,813 @@
...
+    "before": "до",
+    "after": "после",
+    "add at position": "Добавить {position} <code>{task}</code>",
+    "create first task": "Создайте ваш первый task",
+    "dynamic": "Динамический",
+    "choice": "Выбор",
+    "sequential": "Последовательный",
+    "can not delete": "Невозможно удалить",
+    "can not have less than 1 task": "Каждый flow должен иметь хотя бы один task.",
+    "task id already exists": "Task Id уже существует",
```
Comment (smunteankestra) [inline ui/src/translations/ru.json]:
```suggestion
    "task id already exists": "ID задачи уже существует",
```

File: ui/src/translations/ru.json [hunk 38]
```diff
@@ -0,0 +1,813 @@
...
+    "error in editor": "В редакторе найдена ошибка",
+    "delete task confirm": "Вы хотите удалить task <code>{taskId}</code>?",
+    "can not save": "Невозможно сохранить",
+    "flow must have id and namespace": "Flow должен иметь id и namespace.",
+    "readonly property": "Свойство только для чтения",
+    "namespace and id readonly": "Свойства `namespace` и `id` не могут быть изменены — они установлены на начальные значения. Если вы хотите переименовать flow или изменить его namespace, вы можете создать новый flow и удалить старый.",
+    "avg": "Среднее",
+    "sum": "Сумма",
+    "min": "Min",
+    "max": "Max",
```
Comment (smunteankestra) [inline ui/src/translations/ru.json]:
```suggestion
    "max": "Макс",
```

File: ui/src/translations/ru.json [hunk 39]
```diff
@@ -0,0 +1,813 @@
...
+    "editor": "Редактор",
+    "error in editor": "В редакторе найдена ошибка",
+    "delete task confirm": "Вы хотите удалить task <code>{taskId}</code>?",
+    "can not save": "Невозможно сохранить",
+    "flow must have id and namespace": "Flow должен иметь id и namespace.",
+    "readonly property": "Свойство только для чтения",
+    "namespace and id readonly": "Свойства `namespace` и `id` не могут быть изменены — они установлены на начальные значения. Если вы хотите переименовать flow или изменить его namespace, вы можете создать новый flow и удалить старый.",
+    "avg": "Среднее",
+    "sum": "Сумма",
+    "min": "Min",
```
Comment (smunteankestra) [inline ui/src/translations/ru.json]:
```suggestion
    "min": "Мин",
```

File: ui/src/translations/ru.json [hunk 40]
```diff
@@ -0,0 +1,813 @@
...
+    "slack support": "Задайте любой вопрос через Slack",
+    "join community": "Присоединяйтесь к сообществу",
+    "reach us": "Свяжитесь с нами",
+    "new version": "Доступна новая версия {version}!",
+    "error detected": "Обнаружены ошибки",
+    "warning detected": "Обнаружены предупреждения",
+    "cannot swap tasks": "Невозможно поменять местами tasks",
+    "preview": "Предварительный просмотр",
+    "open": "Открыть",
+    "dependency task": "Task {taskId} является зависимостью другого task",
```
Comment (smunteankestra) [inline ui/src/translations/ru.json]:
```suggestion
    "dependency task": "Задача {taskId} является зависимостью другого задачи",
```

File: ui/src/translations/ru.json [hunk 41]
```diff
@@ -0,0 +1,813 @@
...
+      "warning": "Это может привести к параллельным выполнениям для одного и того же trigger и должно рассматриваться как крайняя мера.",
+      "button": "Разблокировать trigger",
+      "success": "Trigger разблокирован"
+    },
+    "restart trigger": {
+      "button": "Перезапустить trigger"
+    },
+    "date format": "Формат даты",
+    "timezone": "Часовой пояс",
+    "add task": "Добавить task",
```
Comment (smunteankestra) [inline ui/src/translations/ru.json]:
```suggestion
    "add task": "Добавить задачу",
```

File: ui/src/translations/ru.json [hunk 42]
```diff
@@ -0,0 +1,813 @@
...
+        "folder_deletion_description": "Вы уверены, что хотите удалить эту папку и все ее содержимое?",
+        "confirm": "Подтвердить"
+      },
+      "path": {
+        "copy": "Скопировать путь",
+        "success": "Путь скопирован в буфер обмена",
+        "error": "Произошли ошибки при копировании пути в буфер обмена"
+      }
+    },
+    "continue backfill": "Продолжить backfill",
```
Comment (smunteankestra) [inline ui/src/translations/ru.json]:
```suggestion
    "continue backfill": "Продолжить заполнение",
```

File: ui/src/translations/ru.json [hunk 43]
```diff
@@ -0,0 +1,813 @@
...
+    "delete backfill": "Удалить backfill",
```
Comment (smunteankestra) [inline ui/src/translations/ru.json:608]:
```suggestion
    "delete backfill": "Удалить заполнение",
```

File: ui/src/translations/ru.json [hunk 44]
```diff
@@ -0,0 +1,813 @@
...
+    "pause backfill": "Приостановить backfill",
```
Comment (smunteankestra) [inline ui/src/translations/ru.json]:
```suggestion
    "pause backfill": "Приостановить заполнение",
```

File: ui/src/translations/ru.json [hunk 45]
```diff
@@ -0,0 +1,813 @@
...
+    "continue backfills": "Продолжить backfills",
```
Comment (smunteankestra) [inline ui/src/translations/ru.json]:
```suggestion
    "continue backfills": "Продолжить заполнения",
```

File: ui/src/translations/ru.json [hunk 46]
```diff
@@ -0,0 +1,813 @@
...
+    "delete backfills": "Удалить backfills",
```
Comment (smunteankestra) [inline ui/src/translations/ru.json]:
```suggestion
    "delete backfills": "Удалить заполнения",
```

File: ui/src/translations/ru.json [hunk 47]
```diff
@@ -0,0 +1,813 @@
...
+    "pause backfills": "Приостановить backfills",
```
Comment (smunteankestra) [inline ui/src/translations/ru.json]:
```suggestion
    "pause backfills": "Приостановить заполнения",
```

File: ui/src/translations/ru.json [hunk 48]
```diff
@@ -0,0 +1,813 @@
...
+    "backfill executions": "Выполнения backfill",
```
Comment (smunteankestra) [inline ui/src/translations/ru.json]:
```suggestion
    "backfill executions": "Выполнения заполнение",
```

File: ui/src/translations/ru.json [hunk 49]
```diff
@@ -0,0 +1,813 @@
...
+    "execute backfill": "Выполнить backfill",
```
Comment (smunteankestra) [inline ui/src/translations/ru.json]:
```suggestion
    "execute backfill": "Выполнить заполнение",
```

File: ui/src/translations/ru.json [hunk 50]
```diff
@@ -0,0 +1,813 @@
...
+    "relative": "Относительный",
+    "relative start date": "Относительная дата начала",
+    "relative end date": "Относительная дата окончания",
+    "now": "Сейчас",
+    "absolute": "Абсолютный",
+    "backfill": "Backfill",
```
Comment (smunteankestra) [inline ui/src/translations/ru.json]:
```suggestion
    "backfill": "Заполнение",
```

File: ui/src/translations/ru.json [hunk 51]
```diff
@@ -0,0 +1,813 @@
...
+      "need-help": {
+        "title": "Нужна помощь?"
+      }
+    },
+    "pluginPage": {
+      "title1": "Плагины — это строительные блоки для ваших рабочих процессов.",
+      "title2": "Ищите tasks и triggers для создания вашего flow.",
+      "search": "Поиск среди +{count} плагинов"
+    },
+    "no inputs": "Этот flow не имеет inputs.",
```
Comment (smunteankestra) [inline ui/src/translations/ru.json]:
```suggestion
    "no inputs": "Этот flow не имеет входных данных.",
```

File: ui/src/translations/ru.json [hunk 52]
```diff
@@ -0,0 +1,813 @@
...
+    "settings": {
+      "label": "Настройки",
+      "blocks": {
+        "configuration": {
+          "label": "Основная конфигурация",
+          "fields": {
+            "language": "Язык",
+            "default_namespace": "Namespace по умолчанию",
+            "log_level": "Уровень Log по умолчанию",
```
Comment (smunteankestra) [inline ui/src/translations/ru.json]:
```suggestion
            "log_level": "Уровень логов по умолчанию",
```

File: ui/src/translations/ru.json [hunk 53]
```diff
@@ -0,0 +1,813 @@
...
+            "log_display": "Отображение Log по умолчанию",
```
Comment (smunteankestra) [inline ui/src/translations/ru.json]:
```suggestion
            "log_display": "Отображение логов по умолчанию",
```

File: ui/src/translations/ru.json [hunk 54]
```diff
@@ -0,0 +1,813 @@
...
+        "6": {
+          "title": "Все готово!",
+          "content": "Проверьте вкладки Logs и Outputs, чтобы отслеживать прогресс выполнения.<br /><br />Затем перейдите на страницу Flows, чтобы изучить другие варианты использования.",
+          "primary": "Создать новый flow",
+          "secondary": "Исследовать другие Flows"
+        }
+      }
+    },
+    "enable concurrency": "Включить concurrency",
+    "auditlogs": "Audit Logs",
```
Comment (smunteankestra) [inline ui/src/translations/ru.json]:
```suggestion
    "auditlogs": "Аудиторские логи",
```

File: ui/src/translations/ru.json [hunk 55]
```diff
@@ -0,0 +1,813 @@
...
+    "iam": "IAM",
+    "tenant": "Tenant",
+    "tenants": "Tenants",
+    "ee-tooltip": {
+      "features-blocked": "Эта функция требует Enterprise Edition.",
+      "button": "Свяжитесь с нами"
+    },
+    "flow_only": "Доступно только на вкладке Flow.",
+    "bulk unpause backfills": "Возобновить {count} backfill",
```
Comment (smunteankestra) [inline ui/src/translations/ru.json]:
```suggestion
    "bulk unpause backfills": "Возобновить {count} заполнение",
```

File: ui/src/translations/ru.json [hunk 56]
```diff
@@ -0,0 +1,813 @@
...
+    "bulk pause backfills": "Приостановить {count} backfill",
```
Comment (smunteankestra) [inline ui/src/translations/ru.json]:
```suggestion
    "bulk pause backfills": "Приостановить {count} заполнение",
```

File: ui/src/translations/ru.json [hunk 57]
```diff
@@ -0,0 +1,813 @@
...
+    "bulk delete backfills": "Удалить {count} backfill",
```
Comment (smunteankestra) [inline ui/src/translations/ru.json]:
```suggestion
    "bulk delete backfills": "Удалить {count} заполнение",
```

File: ui/src/translations/ru.json [hunk 58]
```diff
@@ -0,0 +1,813 @@
...
+    "bulk unlock": "Разблокировать {count} triggers",
+    "bulk disabled status": {
+      "true": "Отключить {count} triggers",
+      "false": "Включить {count} triggers"
+    },
+    "unlock": "Разблокировать",
+    "bulk success unpause backfills": "{count} backfills возобновлены",
```
Comment (smunteankestra) [inline ui/src/translations/ru.json]:
```suggestion
    "bulk success unpause backfills": "{count} заполнения возобновлены",
```

File: ui/src/translations/ru.json [hunk 59]
```diff
@@ -0,0 +1,813 @@
...
+    "bulk success pause backfills": "{count} backfills приостановлены",
```
Comment (smunteankestra) [inline ui/src/translations/ru.json]:
```suggestion
    "bulk success pause backfills": "{count} заполнения приостановлены",
```

File: ui/src/translations/ru.json [hunk 60]
```diff
@@ -0,0 +1,813 @@
...
+    "bulk success delete backfills": "{count} backfills удалены",
```
Comment (smunteankestra) [inline ui/src/translations/ru.json]:
```suggestion
    "bulk success delete backfills": "{count} backfills заполнения",
```

File: ui/src/translations/ru.json [hunk 61]
```diff
@@ -0,0 +1,813 @@
...
+    "bulk success unlock": "{count} triggers разблокированы",
+    "bulk success disabled status": {
+      "true": "{count} trigger(s) отключены",
+      "false": "{count} trigger(s) включены"
+    },
+    "no_namespaces": "Ни один namespace не соответствует критериям поиска.",
+    "plugin defaults": "Параметры по умолчанию для плагина",
+    "secret": {
+      "names": "Secrets",
```
Comment (smunteankestra) [inline ui/src/translations/ru.json]:
```suggestion
      "names": "Cекреты",
```

File: ui/src/translations/ru.json [hunk 62]
```diff
@@ -0,0 +1,813 @@
...
+      "inherited": "Унаследованные secrets"
```
Comment (smunteankestra) [inline ui/src/translations/ru.json]:
```suggestion
      "inherited": "Унаследованные секреты"
```

File: ui/src/translations/ru.json [hunk 63]
```diff
@@ -0,0 +1,813 @@
...
+      "note": "Обратите внимание, что для типов input SECRET и FILE команда должна быть адаптирована для соответствия реальным значениям."
+    },
+    "logs_view": {
+      "raw": "Временной вид",
+      "raw_details": "Показать полные task logs и flow logs в формате необработанных временных меток",
+      "compact": "Вид по умолчанию",
+      "compact_details": "Показать task logs в компактном виде, сгруппированном по каждой task"
+    },
+    "execution_deletion": {
+      "logs": "Удалить logs",
```
Comment (smunteankestra) [inline ui/src/translations/ru.json]:
```suggestion
      "logs": "Удалить логи",
```

File: ui/src/translations/ru.json [hunk 64]
```diff
@@ -0,0 +1,813 @@
...
+        "message": "Вы можете создать или импортировать файлы namespace из боковой панели и посмотреть вводное видео для нашего редактора."
+      }
+    },
+    "curl": {
+      "command": "Команда cURL",
+      "note": "Обратите внимание, что для типов input SECRET и FILE команда должна быть адаптирована для соответствия реальным значениям."
+    },
+    "logs_view": {
+      "raw": "Временной вид",
+      "raw_details": "Показать полные task logs и flow logs в формате необработанных временных меток",
```
Comment (smunteankestra) [inline ui/src/translations/ru.json]:
```suggestion
      "raw_details": "Показать полные task logs и flow логи в формате необработанных временных меток",
```

File: ui/src/translations/ru.json [hunk 65]
```diff
@@ -0,0 +1,813 @@
...
+      "compact": "Вид по умолчанию",
+      "compact_details": "Показать task logs в компактном виде, сгруппированном по каждой task"
```
Comment (smunteankestra) [inline ui/src/translations/ru.json]:
```suggestion
      "compact_details": "Показать task логи в компактном виде, сгруппированном по каждой задаче"
```

**Pull request merged**